src/mappers/mapToData.spec.js

```diff
@@ -1,31 +1,86 @@
 /* eslint-env jest */
 import tester from 'trampss-redux-saga-tester'
+import 'whatwg-fetch'
```

Comment:
Why did you choose that ?
Why do you need that into `mapToData` ?
Is it for `Headers` ?

Reply:
Yes for make Headers, you know another method ?

Reply:
Try with `Map`: https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Map

You can see an example down the page linked above

Reply:
:clap: